roadmap/2022.md

```diff
@@ -54,3 +54,29 @@ Misc:
   - [ ] Finish changes regarding the API keys
   - [ ] Finish changes regarding the new index routes
   - [ ] Add the tenant tokens methods
+
+## Q2
+
+Analytics:
+- [ ] Finish the implementation in the remaining SDKs (Java, Javascript, Rust)
+- [ ] Plan how we will implement the analytics on the integrations, tools & plugins.
+- [ ] _(may not be required)_ Define the specs to support analytics in some tools with core team.
+- [ ] Implement it on the integrations.
+- [ ] Implement it on the plugins.
+
+Processes:
+- [ ] Change the Integrations' Team developers scope.
+
+Improvements:
+- [ ] Add a helpful docker/compose setup in every integration.
+- [ ] Code coverage:
+  - [ ] Define the tool which will be used (codecov, coveralls, others).
+  - [ ] Implement it in the SDKs.
+  - [ ] Implement it on the integrations which makes sense to have coverage.
+
+Meilisearch releases:
```

Comment:
Don't forget v0.28.0. Even if you cannot add a lot of details, you can say the routes will be stabilized (regarding naming and behavior) and will involve a lot of work for the integration team

Reply:
I've added some items about it with a total of 43% of confidence 😆 ! Check it out, please :)